go/base/tests/test_go_start_conversation.py

```diff
@@ -18,8 +18,6 @@ def setUp(self):
         self.command.stderr = StringIO()
 
     def test_sanity_checks(self):
-        self.assertRaisesRegexp(CommandError, 'provide --email-address',
-            self.command.handle, email_address=None, conversation_key=None)
```

Comment:
I can but I removed the test because it would now test `BaseGoCommand#mk_user_api()`, not anything in `go_start_conversation`.